app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp

```diff
@@ -15,41 +17,54 @@
 
 <script defer src="${pageContext.request.contextPath}/resources/js-bundles/geneSearchForm.bundle.js"></script>
 <script>
-    document.addEventListener("DOMContentLoaded", function(event) {
-      geneSearchForm.render({
-        host: '${pageContext.request.contextPath}/',
-        resource: 'json/suggestions/species',
-        wrapperClassName: 'row expanded',
-        actionEndpoint: 'search',
+    document.addEventListener("DOMContentLoaded", function (event) {
+        geneSearchForm.render({
+            host: '${pageContext.request.contextPath}/',
+            resource: 'json/suggestions/species',
+            wrapperClassName: 'row expanded',
+            actionEndpoint: 'search',
 
-        autocompleteClassName: 'small-12 medium-8 columns',
-        suggesterEndpoint: 'json/suggestions/gene_ids',
+            autocompleteClassName: 'small-12 medium-8 columns',
+            suggesterEndpoint: 'json/suggestions/gene_ids',
 
-        enableSpeciesSelect: true,
-        speciesSelectClassName: 'small-12 medium-4 columns',
+            enableSpeciesSelect: true,
+            speciesSelectClassName: 'small-12 medium-4 columns',
 
-        searchExamples: [
-          {
-            text: 'CFTR (gene symbol)',
-            url: '${pageContext.request.contextPath}/search?symbol=CFTR'
-          },
-          {
-            text: 'ENSG00000115904 (Ensembl ID)',
-            url: '${pageContext.request.contextPath}/search?ensgene=ENSG00000115904'
-          },
-          {
-            text: '657 (Entrez ID)',
-            url: '${pageContext.request.contextPath}/search?entrezgene=657'
-          },
-          {
-            text: 'MGI:98354 (MGI ID)',
-            url: '${pageContext.request.contextPath}/search?mgi_id=MGI:98354'
-          },
-          {
-            text: 'FBgn0004647 (FlyBase ID)',
-            url: '${pageContext.request.contextPath}/search?flybase_gene_id=FBgn0004647'
-          }
-        ]
-      }, 'search-form')
+            autocompleteLabel: ``,
```

Comment:
Should these be single quotes? I am not sure.

```suggestion
            autocompleteLabel: '',
```

Reply:
In our React packages we use [backtick strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) but in our JSPs we should use old-style JavaScript, with either double or single quotes. We chose the latter.